lib/ruby_lsp/ruby_lsp_rails/hover.rb

```diff
@@ -91,8 +93,17 @@ def generate_column_content(name)
         return if model.nil?
 
         schema_file = model[:schema_file]
+        if schema_file
+          location = @schema_collector.tables[model[:schema_table]]
+          fragment = "L#{location.start_line},#{location.start_column}-"\
+            "#{location.end_line},#{location.end_column}" if location
```

Comment:
Line information in Prism is 1 based, so we need to subtract one.
```suggestion
          fragment = "L#{location.start_line - 1},#{location.start_column}-"\
            "#{location.end_line - 1},#{location.end_column}" if location
```